backend/api/migrations/0012_update_signing_authority_declaration_statement.py

```diff
@@ -0,0 +1,38 @@
+import logging
+from django.db import migrations
+
+def update_sign_auth_assertion(apps, schema_editor):
+    """
+    Updates the signing authority declaration statement
+
+    Previous label:
+    "I confirm that records evidencing each matter reported under section 11.11 (2) of the 
+    Regulation are available on request."
+
+    New label:
+    "I confirm that records evidencing each matter reported under section 17 of the Low Carbon
+    Fuel (General) Regulation are available on request."
+    """
+    signing_authority_assertion = apps.get_model('api', 'SigningAuthorityAssertion')
+    try:
+        assertion = signing_authority_assertion.objects.get(id=1)

```

Comment:
Are we certain that id=1 is the correct assertion here? Is there a more consistent way to get the correct assertion across environments?

Reply:
You are right, Alex! Using record IDs isn't a good practice because they can change in different environments. I was unable to find any unique names for the `SigningAuthorityAssertion` records in the code. Would you like me to add a unique name to each record? Then I could update this migration file to use these unique names instead of the automatically created IDs.

Another solution would be to find the records by their text (description). This isn't ideal, but it is still better than updating the records by their IDs.

Reply:
hmm yeah a bit tricky if there isn't already a different unique field, and selecting by long form text isn't idea. Let's just continue with your id solution but please confirm that each environment uses those ids for our target records.

Reply:
I can confirm that all the environments (test, development, and production) use the same IDs for our target records.